lib/style.css

```diff
@@ -996,3 +996,8 @@ h4.line:after {
 .route-type-select .toggle.expanded::after {
   content: "\f068";
 }
+
+/* Datatools backend exception in popover overflows the container */
+.popover-content {
+  overflow-x: scroll;
```

Comment:
Is there any way to increase the size of the popup? Otherwise I guess scrolling is alright as well. I think the scrollbar will appear by default right

Reply:
The size of the popup could be expanded, but personally I find the scrollbar to be the best of both worlds - cleaner display while maintaining option for the user (generally us) to investigate details.